synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -165,8 +165,19 @@ Dialog_Guide::on_ok_or_apply_pressed(bool ok)
 		curr_guide->angle = synfig::Angle::rad(angle_widget->get_value());
 	}
 
-	curr_guide->point[0] = x_widget->get_value()/x_factor;
-	curr_guide->point[1] = y_widget->get_value()/y_factor;
+	switch (App::distance_system)
+	{
+		case 0: //units system
+			curr_guide->point[0] = x_widget->get_value();
+			curr_guide->point[1] = y_widget->get_value();
+			break;
+		case 1: //pixel system
```

Comment:
Instead of using integer literals, use enum "named" items:
https://github.com/synfig/synfig/blob/12d62c42c8e31ae99b925badb84e0fee2f3a27ec/synfig-core/src/synfig/distance.h#L46-L64